modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -2948,6 +2948,115 @@ definition {
 			locator1 = "ObjectAdmin#VIEW_ACTIONS_LABEL");
 	}
 
+	@description = "LPS-165849 - Verify terms from a related custom object (child) are replaced correctly in an email notification"
+	@priority = 5
+	test CanSendEmailNotificationWithTermsComingFromCustomObjectRelationship {
+		property custom.properties = "feature.flag.LPS-165849=true";
+		property custom.properties = "feature.flag.LPS-171625=true";
```

Comment:
Please just remove the following line: **_property custom.properties = "feature.flag.LPS-171625=true";_**. The FF was removed, see [here](https://issues.liferay.com/browse/LPS-179736)